### ISSUE TITLE
How to set learning rate when you use multiple gpus?

### ISSUE THREAD

**Issue by hanjianhua44** [traveller59/second.pytorch#278]
The default learning rate set in config file is 3e-3, I want to know when I train on 8gpus, should I reset the learning rate to 0.024 (8*3e-3)?

**Comment by turboxin** [comment]
same question
